actions/image_partition_action.go

```diff
@@ -81,6 +82,10 @@ should be mounted.
 Optional properties:
 
 - options -- list of options to be added to appropriate entry in fstab file.
+- buildtime -- if set to true then mount point is added only during the image
+creation without additional entry in `/etc/fstab' on target. Mount point directory
+is wiped out from the image.
+Defaults to false.
```

Comment:
The removal seems a bit unexpected. May make sense to document this a bit better and provide some hint on how to do it (e.g. use `/media/temporarybadger` not `/media` or `/mnt` directly)

I've also pondered of whether instead of the user having to specifiy a mountpoint in these cases that should be left unset and debos will create one; but that then needs to communicate to user scripts (or overlays) the location and that seems more haphazard..

So I *think* this way is probably the best trade-off; but a bit more clear documentation will be good.